src/util/extractCursorFromNextUri.ts

```diff
@@ -0,0 +1,15 @@
+import { URL } from 'url';
+
+/*
+ * Extracts cursor 00ubfjQEMYBLRUWIEDKK from
+ * https://lifeomic.okta.com/api/v1/users?after=00ubfjQEMYBLRUWIEDKK
+ */
+export default function extractCursorFromNextUri(
```

Comment:
I suspect this won't be required for the `iterate*` functions, but we'll see. It's only referenced in the current iterative fetch functions for figuring out whether to iterate more I think.

Reply:
Removed